### DISCUSSION TITLE
How to switch chains

### DISCUSSION THREAD

**Discussion by StevenBarnett1** [WalletConnect/walletconnect-monorepo#2665] [Developer Support]
Hey guys so i have walletconnect connecting to alpha wallet on 5 different chains simultaneously and im successfully establishing the connection. But how do i specify which chain i want to be on after that once the connection is established and i want to send a transaction?

**Comment by glitch-txs** [comment]
Hi which library are you using?

**Comment by StevenBarnett1** [comment]
I’m using ethers, web3modal, and react native

**Reply by glitch-txs** [reply]
Then I assume the package for react native right?
You can try with
```ts
const chainId = 1 //for example for mainnet
provider.setDefaultChain(`eip155:${chainId}`)
```

If I'm not mistaken it follows universal-provider rules
https://docs.walletconnect.com/2.0/web/providers/universal#multi-chain
In said case you will need to set the default chain before passing it to ethers I think 🤔
